src/org/parosproxy/paros/core/scanner/ScannerParam.java

```diff
@@ -213,6 +233,8 @@ protected void parse() {
 
         this.scanHeadersAllRequests = getBoolean(SCAN_HEADERS_ALL_REQUESTS, false);
 
+        this.addQueryParam = getBoolean(SCAN_ADD_QUERY_PARAM, false);

```

Comment:
Wouldnt it be better to have this as on by default to maintain the current behaviour?

Reply:
I don't mind changing it, I went false due to the feedback/attention it's been getting.

Reply:
Anyone else got feedback on this? Shall I start a User Group thread?

Reply:
I'm OK changing the default given the reported FP rate. We could mention the change in the release notes, to be more noticeable.

Reply:
Agreed, does that seems reasonable to you @psiinon?

Reply:
Help PR updated: https://github.com/zaproxy/zap-core-help/pull/203

Reply:
Yeah, but we should def mention this in the release notes.